strava_local_client.py

```diff
@@ -48,7 +48,7 @@ def auth_callback():
         auth_url = API_CLIENT.authorization_url(
             client_id=args['<client_id>'],
             redirect_uri='http://127.0.0.1:{port}/auth'.format(port=args['--port']),
-            scope='view_private,write',
+            scope=['activity:write','activity:read_all','profile:read_all','profile:write','read_all'],
```

Comment:
I got lazy trying to map the new api permissions so I just used all of them. can probably figure out which ones are really needed if necessary